services/catalog/src/simcore_service_catalog/schemas/project.py

```diff
@@ -0,0 +1,84 @@
+# From  project-v0.0.1.json
+# from __future__ import annotations TODO: ???
+
+# TODO: why pylint error in pydantic???
+# pylint: disable=no-name-in-module
+
+import sys
+from enum import Enum
+from pathlib import Path
+from typing import Dict, List, Optional, Union
+
+from pydantic import BaseModel, EmailStr, Field
+
+current_file = Path(sys.argv[0] if __name__ == "__main__" else __file__).resolve()
+
+KEY_RE = r"^(simcore)/(services)(/demodec)?/(comp|dynamic|frontend)(/[^\s]+)+$"
```

Comment:
remove demodec
frontend I'm not sure that is still existing?

Reply:
For the moment i keep compatibility with projects.yaml ...